### DISCUSSION TITLE
"Finally" & "Let's be honest" Linked videos

### DISCUSSION THREAD

**Discussion by nob-rot** [ThioJoe/YT-Spam-Lists#170] [General]
I started a list of videos that are linked with "Finally" & "Let's be honest" bots
https://github.com/ThioJoe/YT-Spam-Lists/issues/169

Is this format ok? It will be a ongoing list so should I edit the OP with a timestamp or should I post as a reply?

**Comment by Firecul** [comment]
**[Obviously, go with what ThioJoes says over me]**

I'm ok, personally, with the additions as long as you stop once those ids are added and that issue closed naturally, at that point you can always make another issue for more.
I'll get round to checking and adding those asap (tomorrow most likely.)

**Comment by ThioJoe** [comment]
Yea that's all fine. Like Firecul said just make a new issue once the videos get added and the issue closed. But if they haven't been added yet you can just edit and add more to it. Not super crucial to timestamp it

**Comment by nob-rot** [comment]
Thanks for the input, it seems, despite how prevalent "let's be honest", "last fight", "finally" bots are they only promote a small number of channels, I've seen copycats but their occurrence looks like low tech [manual]
